docker-compose.yaml

```diff
@@ -1,26 +1,27 @@
 version: '3'
 
 services:
+
   peatio:
     build:
       context: peatio
       dockerfile: Dockerfile
+    restart: always
     depends_on:
       - db
+      - redis
+      - rabbitmq
     ports:
-      - "8000:80"
-    restart: always
+    - "80:8080"
     environment:
       RAILS_ENV: development
 
   db:
-    image: mysql:5.7
-    volumes:
-      - db_data:/var/lib/mysql
```

Comment:
for development purpose we need the database, because developer should very rarely reset db